plugins/LilyPad/DirectInput.cpp

```diff
@@ -220,7 +222,7 @@ class DirectInputDevice : public Device
             format.dwFlags = 0;
             format.dwNumObjs = numPhysicalControls;
             format.rgodf = formats;
-            int res = did->SetDataFormat(&format);
+            // int res = did->SetDataFormat(&format); // is not used
```

Comment:
Yeah, `did->SetDataFormat()` is required. I should've only removed the `int rest = `.